### ISSUE TITLE
Ignore :w !sudo

### ISSUE THREAD

**Issue by dhnam** [hrsh7th/cmp-cmdline#109]
I forgot to use sudo to edit file, I searched for how to save in super user permission, and told to use `:w !sudo tee %`.

I tried, and when I type '!' after :w, neovim stopped.

Guess cmp-cmdline tries to load autocomplete of ! after :w, even though I use default setting.

**Comment by theSprog** [comment]
Yes, I also encountered the problem you mentioned, but when I was about to type `r !ls`, neovim froze after I typed `!`. 

But one thing I need to add is that neovim is not stopped, it is just stuck in a long processing logic. After I waited for about a few minutes, neovim still successfully read the contents of ls. , I tried disabling cmp-cmdline and the bug disappeared, so I think it should be a problem with cmp-cmdline

**Comment by theSprog** [comment]
I opened another issue to discuss this issue. #112

**Comment by mau-mauricelim** [comment]
I have the same issue with `:w !` and `:r !`
